pkg/sql/logictest/tests/fakedist/fakedist_test: TestLogic_zone_config_system_tenant failed

pkg/sql/logictest/tests/fakedist/fakedist_test.TestLogic_zone_config_system_tenant [failed](https://tanzanite.cluster.engflow.com/invocations/default/e264ea70-67cb-43c4-b753-03da30923a39?testReportRun=3&testReportShard=24&testReportAttempt=1#targets-Ly9wa2cvc3FsL2xvZ2ljdGVzdC90ZXN0cy9mYWtlZGlzdDpmYWtlZGlzdF90ZXN0) on master @ [7571549d8772a087ab4577d1e770e02582ba5877](https://github.com/cockroachdb/cockroach/commits/7571549d8772a087ab4577d1e770e02582ba5877):


```
=== RUN   TestLogic_zone_config_system_tenant
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestLogic_zone_config_system_tenant1995980402
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:144: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
[10:47:04] setting distsql_workmem='29065B';
    logic.go:4371: -- test log scope end --
test logs left over in: outputs.zip/logTestLogic_zone_config_system_tenant1995980402
--- FAIL: TestLogic_zone_config_system_tenant (67.87s)
=== RUN   TestLogic_zone_config_system_tenant/dropped_relation_gc_ttl
[10:47:09] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant: 31 statements
[10:47:12] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant: 33 statements
    logic.go:3012: 
         
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant:198: SELECT
          crdb_internal.pretty_key(start_key, -1),
          crdb_internal.pb_to_json('cockroach.roachpb.SpanConfig', config)->'gcPolicy'->>'ttlSeconds'
        FROM system.span_configurations
        WHERE start_key >= (SELECT crdb_internal.table_span(100)[1])
        ORDER BY start_key
        expected:
            /Table/106  14400
            /Table/107  14400
            /Table/110  90001
            /Table/111  90001
            /Table/112  14400
            
        but found (query options: "retry") :
            /Table/106  14400
            /Table/107  14400
            /Table/110  90001
            /Table/111  1
            /Table/112  1
            
[10:47:59] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant: 35 statements
[10:47:59] --- done: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant with config fakedist: 35 tests, 1 failures
[10:48:04] --- total progress: 35 statements
--- total: 35 tests, 1 failures
    --- FAIL: TestLogic_zone_config_system_tenant/dropped_relation_gc_ttl (52.46s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=3</code>
 - <code>shard=24</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_zone_config_system_tenant.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-42039

pkg/sql/logictest/tests/fakedist/fakedist_test.TestLogic_zone_config_system_tenant [failed](https://tanzanite.cluster.engflow.com/invocations/default/0ba9a8d8-3d04-455d-b066-a74a801783e2?testReportRun=2&testReportShard=24&testReportAttempt=1#targets-Ly9wa2cvc3FsL2xvZ2ljdGVzdC90ZXN0cy9mYWtlZGlzdDpmYWtlZGlzdF90ZXN0) on master @ [31794d8d89e3e117e1654cb13ad109da04de5445](https://github.com/cockroachdb/cockroach/commits/31794d8d89e3e117e1654cb13ad109da04de5445):


```
=== RUN   TestLogic_zone_config_system_tenant
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestLogic_zone_config_system_tenant586894767
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:144: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
    logic.go:4371: -- test log scope end --
test logs left over in: outputs.zip/logTestLogic_zone_config_system_tenant586894767
--- FAIL: TestLogic_zone_config_system_tenant (67.75s)
=== RUN   TestLogic_zone_config_system_tenant/dropped_relation_gc_ttl
[10:47:43] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant: 31 statements
[10:47:46] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant: 33 statements
    logic.go:3012: 
         
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant:198: SELECT
          crdb_internal.pretty_key(start_key, -1),
          crdb_internal.pb_to_json('cockroach.roachpb.SpanConfig', config)->'gcPolicy'->>'ttlSeconds'
        FROM system.span_configurations
        WHERE start_key >= (SELECT crdb_internal.table_span(100)[1])
        ORDER BY start_key
        expected:
            /Table/106  14400
            /Table/107  14400
            /Table/110  90001
            /Table/111  90001
            /Table/112  14400
            
        but found (query options: "retry") :
            /Table/106  14400
            /Table/107  14400
            /Table/110  90001
            /Table/111  1
            /Table/112  1
            
[10:48:32] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant: 35 statements
[10:48:32] --- done: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant with config fakedist: 35 tests, 1 failures
[10:48:38] --- total progress: 35 statements
--- total: 35 tests, 1 failures
    --- FAIL: TestLogic_zone_config_system_tenant/dropped_relation_gc_ttl (52.43s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=24</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_zone_config_system_tenant.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

pkg/sql/logictest/tests/fakedist/fakedist_test.TestLogic_zone_config_system_tenant [failed](https://tanzanite.cluster.engflow.com/invocations/default/681bf7cd-f8c6-43f4-b47b-b7769019fea7?testReportRun=2&testReportShard=25&testReportAttempt=1#targets-Ly9wa2cvc3FsL2xvZ2ljdGVzdC90ZXN0cy9mYWtlZGlzdDpmYWtlZGlzdF90ZXN0) on master @ [3f8226eb7f09eb93c26dda4cd47e023a8f56ea23](https://github.com/cockroachdb/cockroach/commits/3f8226eb7f09eb93c26dda4cd47e023a8f56ea23):


```
=== RUN   TestLogic_zone_config_system_tenant
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestLogic_zone_config_system_tenant1952375573
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:144: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
[10:07:49] setting distsql_workmem='53419B';
    logic.go:4371: -- test log scope end --
test logs left over in: outputs.zip/logTestLogic_zone_config_system_tenant1952375573
--- FAIL: TestLogic_zone_config_system_tenant (68.49s)
=== RUN   TestLogic_zone_config_system_tenant/dropped_relation_gc_ttl
[10:07:54] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant: 31 statements
[10:07:56] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant: 32 statements
[10:07:59] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant: 34 statements
    logic.go:3012: 
         
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant:198: SELECT
          crdb_internal.pretty_key(start_key, -1),
          crdb_internal.pb_to_json('cockroach.roachpb.SpanConfig', config)->'gcPolicy'->>'ttlSeconds'
        FROM system.span_configurations
        WHERE start_key >= (SELECT crdb_internal.table_span(100)[1])
        ORDER BY start_key
        expected:
            /Table/106  14400
            /Table/107  14400
            /Table/110  90001
            /Table/111  90001
            /Table/112  14400
            
        but found (query options: "retry") :
            /Table/106  14400
            /Table/107  14400
            /Table/110  90001
            /Table/111  1
            /Table/112  1
            
[10:08:44] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant: 35 statements
[10:08:44] --- done: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant with config fakedist: 35 tests, 1 failures
[10:08:50] --- total progress: 35 statements
--- total: 35 tests, 1 failures
    --- FAIL: TestLogic_zone_config_system_tenant/dropped_relation_gc_ttl (53.42s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=25</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_zone_config_system_tenant.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

pkg/sql/logictest/tests/fakedist/fakedist_test.TestLogic_zone_config_system_tenant [failed](https://tanzanite.cluster.engflow.com/invocations/default/3ac70db1-dc36-4a90-a521-1c8ee348a0bc?testReportRun=2&testReportShard=25&testReportAttempt=1#targets-Ly9wa2cvc3FsL2xvZ2ljdGVzdC90ZXN0cy9mYWtlZGlzdDpmYWtlZGlzdF90ZXN0) on master @ [128fcab4c07413513a05aea1d1494943f4bc3092](https://github.com/cockroachdb/cockroach/commits/128fcab4c07413513a05aea1d1494943f4bc3092):


```
=== RUN   TestLogic_zone_config_system_tenant
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestLogic_zone_config_system_tenant3767271489
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:144: cluster virtualization disabled in global scope due to issue: #76378 (expected label: C-bug)
[09:57:44] setting distsql_workmem='44833B';
    logic.go:4371: -- test log scope end --
test logs left over in: outputs.zip/logTestLogic_zone_config_system_tenant3767271489
--- FAIL: TestLogic_zone_config_system_tenant (68.27s)
=== RUN   TestLogic_zone_config_system_tenant/dropped_relation_gc_ttl
[09:57:49] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant: 31 statements
[09:57:52] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant: 33 statements
    logic.go:3012: 
         
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant:198: SELECT
          crdb_internal.pretty_key(start_key, -1),
          crdb_internal.pb_to_json('cockroach.roachpb.SpanConfig', config)->'gcPolicy'->>'ttlSeconds'
        FROM system.span_configurations
        WHERE start_key >= (SELECT crdb_internal.table_span(100)[1])
        ORDER BY start_key
        expected:
            /Table/106  14400
            /Table/107  14400
            /Table/110  90001
            /Table/111  90001
            /Table/112  14400
            
        but found (query options: "retry") :
            /Table/106  14400
            /Table/107  14400
            /Table/110  90001
            /Table/111  1
            /Table/112  1
            
[09:58:39] --- progress: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant: 35 statements
[09:58:39] --- done: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/sql/logictest/tests/fakedist/fakedist_test_/fakedist_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant with config fakedist: 35 tests, 1 failures
[09:58:45] --- total progress: 35 statements
--- total: 35 tests, 1 failures
    --- FAIL: TestLogic_zone_config_system_tenant/dropped_relation_gc_ttl (52.75s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=25</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLogic_zone_config_system_tenant.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

--- a/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant
+++ b/pkg/sql/logictest/testdata/logic_test/zone_config_system_tenant
@@ -205,8 +205,8 @@ ORDER BY start_key
 /Table/106  14400
 /Table/107  14400
 /Table/110  90001
-/Table/111  90001
-/Table/112  14400
+/Table/111  1
+/Table/112  1
 
 subtest transactional_schemachanges
 

